### PR TITLE
Refuse to add inaccessible folders, warn on inaccessible files during backup

### LINKED ISSUE #916: Verify permissions when adding a new source file or folder
**Describe the bug**

**To Reproduce**
1. Create a new repo with local path.
2. Go to "Sources" and add a directory. Now choose a path you do not have access with the current user, like a different user home dir (`/home/someonelese`). You can select that by pressing enter.

What happens: The whole UI freezes, I cannot click anymore. Note that the path also is not added in the UI.
What should happen: Show an error message.

**Environment (please complete the following information):**
- OS: Fedora 33
- Vorta version: v0.7.5
- Installed from: Flathub

**Additional context**

```
2021-03-06 14:23:09,875 - vorta.i18n - DEBUG - Loading translation succeeded for ['de', 'de-DE', 'de-Latn-DE'].
2021-03-06 14:23:09,893 - apscheduler.scheduler - INFO - Scheduler started
2021-03-06 14:23:12,606 - root - INFO - Using NetworkManagerMonitor NetworkStatusMonitor implementation.
2021-03-06 14:23:12,722 - vorta.borg.borg_thread - INFO - Running command /app/bin/borg --version
2021-03-06 14:23:47,689 - asyncio - DEBUG - Using selector: EpollSelector
2021-03-06 14:24:37,910 - asyncio - DEBUG - Using selector: EpollSelector
2021-03-06 14:24:37,913 - asyncio - DEBUG - Using selector: EpollSelector
2021-03-06 14:24:37,914 - root - DEBUG - Found 0 passwords matching repo URL.
2021-03-06 14:25:51,776 - asyncio - DEBUG - Using selector: EpollSelector
2021-03-06 14:25:51,777 - vorta.borg.borg_thread - DEBUG - Using VortaSecretStorageKeyring keyring to store passwords.
2021-03-06 14:25:51,780 - asyncio - DEBUG - Using selector: EpollSelector
2021-03-06 14:25:51,782 - root - DEBUG - Found 0 passwords matching repo URL.
[…]
2021-03-06 14:25:53,395 - vorta.borg.borg_thread - INFO - Done.
2021-03-06 14:25:53,652 - vorta.borg.borg_thread - WARNING - 
By default repositories initialized with this version will produce security
errors if written to with an older version (up to and including Borg 1.0.8).

If you want to use these older versions, you can disable the check by running:
borg upgrade --disable-tam *************************

See https://borgbackup.readthedocs.io/en/stable/changes.html#pre-1-0-9-manifest-spoofing-vulnerability for details about the security implications.
2021-03-06 14:25:53,655 - vorta.borg.borg_thread - WARNING - 
IMPORTANT: you will need both KEY AND PASSPHRASE to access this repo!
Use "borg key export" to export the key, optionally in printable format.
Write down the passphrase. Store both at safe place(s).

2021-03-06 14:25:53,883 - asyncio - DEBUG - Using selector: EpollSelector
2021-03-06 14:25:54,234 - asyncio - DEBUG - Using selector: EpollSelector
[…]
2021-03-06 14:25:54,245 - asyncio - DEBUG - Using selector: EpollSelector
2021-03-06 14:25:54,252 - root - DEBUG - Found 1 passwords matching repo URL.
2021-03-06 14:25:54,351 - vorta.borg.borg_thread - INFO - Running command /app/bin/borg list --info --log-json --json *************************
```

```
$ flatpak info com.borgbase.Vorta 

Vorta - Backup client

         Kennung: com.borgbase.Vorta
             Ref: app/com.borgbase.Vorta/x86_64/stable
     Architektur: x86_64
           Zweig: stable
         Version: v0.7.5
         License: GPL-3.0
        Ursprung: flathub
        Sammlung: org.flathub.Stable
    Installation: system
     Installiert: 53,7 MB
Laufzeitumgebung: org.kde.Platform/x86_64/5.15
             Sdk: org.kde.Sdk/x86_64/5.15

          Commit: 42c0ff004ec431c3d454072e4311f79fa65e3ba09abc98d07bc1bcca38740…
          Parent: 7bd3aa88cf115575981e8a1e6c947d16f5987b6f8720b6c57470bb1901efa…
         Subject: Update v0.7.5 (b324ae38)
            Date: 2021-03-03 17:42:57 +0000
```

Nothing logged to the Stdout:
```
flatpak run com.borgbase.Vorta                         
QSocketNotifier: Can only be used with threads started with QThread
2021-03-06 14:40:09,791 - vorta.i18n - DEBUG - Loading translation succeeded for ['de', 'de-DE', 'de-Latn-DE'].
2021-03-06 14:40:09,807 - apscheduler.scheduler - INFO - Scheduler started
QObject::connect: No such signal QPlatformNativeInterface::systemTrayWindowChanged(QScreen*)
2021-03-06 14:40:10,003 - root - INFO - Using NetworkManagerMonitor NetworkStatusMonitor implementation.
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
2021-03-06 14:40:10,095 - vorta.borg.borg_thread - INFO - Running command /app/bin/borg --version
```

### PR DESCRIPTION
**Pull request by m3nu**
We now refuse to add unreadable folders and .

warn about access errors during backup

Fixes #916

### CODE REVIEW

File: src/vorta/borg/create.py [hunk 1]
```diff
@@ -32,7 +32,10 @@ def process_result(self, result):
                 repo.total_unique_chunks = stats['total_unique_chunks']
                 repo.save()
 
-            self.app.backup_progress_event.emit(self.tr('Backup finished.'))
+            if result['returncode'] == 1:
+                self.app.backup_progress_event.emit(self.tr('Backup finished. Some files were not accessible.'))
```
Comment (sten0) [inline src/vorta/borg/create.py:36]: I'm not sure that this is correct, because doesn't borg use returncode=1 as a catch-all for a variety of errors?

Comment (m3nu) [reply src/vorta/borg/create.py:36]: You're right. Could also be "Backup finished with warnings. See logs for details."

Upside is it's more "correct". Downside is that the large majority of warnings will be access errors. Can't think of other warnings I've seen. That fact is now hidden in the logs.

Comment (m3nu) [reply src/vorta/borg/create.py:36]: I've added a separate issue #952  to nag Mac users about Full Disk access. This should make such errors very rare and we can use the vague sentence. "Backup finished with warnings. See logs for details."

OK?

Comment (m3nu) [reply src/vorta/borg/create.py:36]: I've now merged the feature to nag Mac users about proper permissions. Mostly due to a overwhelming response on Twitter: https://twitter.com/borg_base/status/1383302657045979144

With this access errors would be fairly rare everywhere and the more vague sentence is likely ok.

**Pull request merged**